std/base.m31

```diff
@@ -12,7 +12,7 @@ handle
   | as_eq _ => None
 end
 
-let (|>) x f = f x
+let (|>) x f : Π α β, α → (α → β) → β = f x
```

Comment:
What do you mean? You want to generalize `|>` to `∏ α, list α → (list α → list α) →
list α`?